### PR TITLE
[9.0][ADD] stock_picking_operation_quick_change: New module to allow change target location in picking operations

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
Thanks.

### CODE REVIEW

**Review by cubells — APPROVED** [review]
Code review and test in runbot.

:+1:

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+
+class StockPickingOperationWizard(models.TransientModel):
+    _name = 'stock.picking.operation.wizard'
+
+    def _prepare_deafault_values(self, picking):
```
Comment (rousseldenis) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard.py]: IMO, even if function names should be at developer choice, avoid spelling faults ('default')

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        return {'location_dest_id': picking.location_dest_id.id}
+
+    @api.model
+    def default_get(self, fields):
+        res = super(StockPickingOperationWizard, self).default_get(fields)
+        active_model = self.env.context['active_model']
+        active_ids = self.env.context['active_ids'] or []
+        picking = self.env[active_model].browse(active_ids)
+        res.update(self._prepare_deafault_values(picking))
```
Comment (rousseldenis) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard.py]: See here above

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review + tested on runbot.
Great usability improvement in real life !
Do you plan v10 too ?

**Comment by sergio-teruel** [comment]
@rousseldenis Changes done.
About the migration, i think that it is a direct migration... (I would have to analyze it)
If you want, you can do it... :smile:

**Review by rousseldenis — APPROVED** [review]
@sergio-teruel Think I'll do it as I need it ;-)

**Comment by JordiBForgeFlow** [comment]
May I ask, what is the use case in real life? Why do you need it?

**Comment by rousseldenis** [comment]
e.g. : you have a transfer to Output Location.
This Location contains several sub-locations.
You want to put your pack in sub-location 1 and have several operations in the picking.
So, to do it as mass assignation is time earned.

**Comment by JordiBForgeFlow** [comment]
Ok, thanks!

**Comment by sergio-teruel** [comment]
@rousseldenis If you start the work, please notify me it.. ok? Thanks...

**Comment by rousseldenis** [comment]
@jbeficent Can you review ?

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot

**Pull request merged**